### PR TITLE
Clarify the scope of the webAppSecurity attributes

### LINKED ISSUE #28326: Documentation update for webAppSecurity attribute
The attributes in the following page 
https://www.ibm.com/docs/en/was-liberty/base?topic=configuration-webappsecurity
applies to the user applications but not to the system applications. 

To clarify, this issue will update 

> Configures web container application security.

to 

> Configures web container application security for user applications only.

### PR DESCRIPTION
**Pull request by una-tapa**
Clarifying the scope of webAppSecurity attributes. 
fixes #28326

### CODE REVIEW

**Comment by una-tapa** [comment]
#build

File: dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 #
 webAppSecurityService.config=Web Container Application Security
-webAppSecurityService.config.desc=Configures web container application security.
+webAppSecurityService.config.desc=Configures web container application security for user applications only.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties]:
```suggestion
webAppSecurityService.config.desc=Configures web container security for user applications only.
```

Comment (dmuelle) [reply dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties]: Will the implied contrast with system applications be clear to users? Or should we be more explicit- like 

> Configures web container security for user applications only. This element is not intended to configure web container security for system applications.

I'm not sure- what do you think @una-tapa ?

Comment (dmuelle) [reply dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties]: or if there is a clear alternative for system applications, we could state that..

> Configures web container security for user applications only. To configure web container security for system applications, [do/use xxxx}.

**Review by dmuelle — COMMENTED** [review]
ID review- had a few questions as I'm not particularly familiar with the context btwn user/system application security

**Comment by una-tapa** [comment]
#libby

**Comment by chirp1** [comment]
"Configures web container security for user applications only." looks good to me.

**Review by dmuelle — APPROVED** [review]

**Pull request merged**